src/custom-widget.ts

```diff
@@ -224,6 +224,10 @@ const openLink: TTagoIO["openLink"] = (url) => {
   sendMessage({ method: "open-link", url });
 };
 
+const closeModal: TTagoIO["closeModal"] = () => {
+  sendMessage({ method: "close-modal" });
+};
```

Comment:
Only parameter required is sending the new message, which is now added to types as well.